src/components/Catalog.js

```diff
@@ -0,0 +1,48 @@
+import { Component } from "react";
+import Movie from './Movie';
+import Rented from './Rented'
+
+class Catalog extends Component {
+
+    constructor() {
+        super();
+        this.state = {
+            search: '',
+            budget: 10
+        } 
+    }
+
+    updateSearch = (e) => {
```

Comment:
you can write it like 
`updateSearch = ({target}) => this.setState({search: target.value})`